rest/rest-server/src/test/java/org/ow2/proactive_grid_cloud_portal/webapp/RestRuntimeExceptionMappingTest.java

```diff
@@ -63,9 +63,9 @@ public void unknown_path_404_exception() throws Exception {
 
         assertEquals(404, exception.getHttpErrorCode());
         assertNotNull(exception.getErrorMessage());
-        assertNotNull(exception.getException());
+        //assertNotNull(exception.getException());
         assertEquals(NotFoundException.class.getName(), exception.getExceptionClass());
-        assertNotNull(exception.getStackTrace());
+        //assertNotNull(exception.getStackTrace());
```

Comment:
You can replace comments by assertNull

Reply:
indeed but this is just to remember it was changed because of a configuration